sklearn/metrics/pairwise.py

```diff
@@ -1187,29 +1187,32 @@ def linear_kernel(X, Y=None, dense_output=True):
 
 
 def polynomial_kernel(X, Y=None, degree=3, gamma=None, coef0=1):
-    """
-    Compute the polynomial kernel between X and Y::
-
+    """Compute the polynomial kernel between X and Y.
         K(X, Y) = (gamma <X, Y> + coef0)^degree
 
     Read more in the :ref:`User Guide <polynomial_kernel>`.
 
     Parameters
     ----------
     X : ndarray of shape (n_samples_X, n_features)
+        A feature array.
 
     Y : ndarray of shape (n_samples_Y, n_features), default=None
+        An optional second feature array. If `None`, uses `Y=X`.
 
     degree : int, default=3
+        Kernel degree.
 
     gamma : float, default=None
-        If None, defaults to 1.0 / n_features.
+        Slope. If None, defaults to 1.0 / n_features.
```

Comment:
I wonder if we can better describe this. Perhaps something like:
```suggestion
        Coefficient of the vector inner product. If None, defaults to 1.0 / n_features.
```

I'm open to opinions on this though. We don't really document `gamma` as anything more than a "parameter" of the kernel in other spots.